examples/runner-docker/variables.tf

```diff
@@ -31,5 +31,6 @@ variable "gitlab_url" {
 }
 
 variable "registration_token" {
+  default = "UkCzuTzyM8rBzs68Hpsu"
```

Comment:
Please can you remove the token, in case a real token would suggest to reset the token.

Reply:
Hi,

It's done,

Thank you very much for your warning.
It was on a fake project that I previously deleted.